src/tokenizer/main.py

```diff
@@ -87,6 +87,13 @@
 )
 group.add_argument("--normalize", help="Normalize punctuation", action="store_true")
 
+parser.add_argument(
+    "--ospl",
```

Comment:
I think these are usually single-letter abbreviations, i.e. "-o".

Reply:
Changed to "--s", as "-o" might confuse users, it is usually for --output.